testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/CLIEmbedHostControllerTestCase.java

```diff
@@ -758,8 +758,27 @@ private String readLogOut() {
     }
 
     private void checkLogging(String line) throws IOException {
-        String logOutput = readLogOut();
-        assertTrue(logOutput, checkLogging(logOutput, line));
+        long delay = System.currentTimeMillis() + 10000;
```

Comment:
Would you mind using `TimeoutUtil.adjust(1000)`? 

That's more useful since it allows us to control all timeouts from a system property and use more time in a slow system.

Reply:
Thanks, fixed.